src/css/modules/progress.css

```
@@ -35,8 +36,12 @@
 }
 
 /* 不同尺寸 */
+.lay-progress.lay-size-xs {
+  --lay-progress-border-radius: 0;
+  height: 2px;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)